src/lib/Exporter/Exporter.php

```diff
@@ -384,17 +351,14 @@ private function generateSubtreeCriteria(?int $mandatorId, ?string $siteAccess):
         $subtreeCriteria = [];
         $rootLocations = $this->siteAccessHelper->getRootLocationsBySiteAccesses($siteAccesses);
         foreach ($rootLocations as $rootLocationId) {
-            $subtreeCriteria[] = new Criterion\Subtree($this->locationService->loadLocation($rootLocationId)->pathString);
+            $subtreeCriteria[] = $this->repository->sudo(function () use ($rootLocationId) {
+                return new Criterion\Subtree($this->locationService->loadLocation($rootLocationId)->pathString);
+            });
```

Comment:
Could rather be something like:
```
$rootLocationList = $this->repository->sudo(function () use ($rootLocations) {
    return $this->locationService->loadLocationList($rootLocations);
});
foreach ($rootLocationList as $rootLocation) {
            $subtreeCriteria[] = new Criterion\Subtree($rootLocation->pathString);
```

Would be faster and somewhat cleaner.
foreach ($rootLocations as $rootLocationId) {
            $subtreeCriteria[] = new Criterion\Subtree($this->locationService->loadLocation($rootLocationId)->pathString);
            $subtreeCriteria[] = $this->repository->sudo(function () use ($rootLocationId) {
                return new Criterion\Subtree($this->locationService->loadLocation($rootLocationId)->pathString);
            });
```